src/core/Header/HeaderLinks.tsx

```diff
@@ -29,23 +28,60 @@ export const HeaderLinks: React.FC<
   searchButton,
   className,
 }) => {
-  const signedIn = sessionState.signedIn;
+  const { signedIn, logOut } = sessionState;
+  const formRef = useRef<HTMLFormElement>(null);
+
   const headerLinkClasses =
     "ui-text-menu2 md:ui-text-menu3 !font-bold py-16 text-neutral-1300 dark:text-neutral-000 md:text-neutral-1000 dark:md:text-neutral-300 hover:text-neutral-1300 dark:hover:text-neutral-000 active:text-neutral-1300 dark:active:text-neutral-000 transition-colors";
 
+  const dropdownMenuLinkClasses =
+    "block p-8 ui-text-menu3 font-semibold text-neutral-1000 dark:text-neutral-300 hover:bg-neutral-100 dark:hover:bg-neutral-1200 active:bg-neutral-200 dark:active:bg-neutral-1100";
```

Comment:
I can't see it explicitly in the design but I would say design would want these links to be as rounded as the dropdown menu itself, so add `rounded-lg`